tutorials/networking/websocket.rst

```diff
@@ -38,59 +38,32 @@ This example will show you how to create a WebSocket connection to a remote serv
 
     extends Node
 
-    # The URL we will connect to
-    export var websocket_url = "wss://libwebsockets.org"
```

Comment:
Why was this removed?

Reply:
Because I tried and couldn't connect to this address

Reply:
The problem is that this doesn't provide an example that can be tried to ensure you've done the right thing, your address below isn't a real address

Reply:
You're right, maybe this should be changed to `var websocket_url = "wss://localhost:8999"`, which would look better. It's best to add a comment # You need to start websocket server that listens to port 8999.

Reply:
I think a proper working server is required, as is provided for the other PR changing this

Reply:
#7645 is IMHO more clean, and predates this one, so I think this should be closed in favor of it